docs/advanced_topics/customisation/custom_user_models.md

```diff
@@ -123,7 +123,8 @@ Replace `wagtail.users` in `settings.INSTALLED_APPS` with the path to `CustomUse
 ```python
 INSTALLED_APPS = [
     ...,
-    "myapp.apps.CustomUsersAppConfig",
+    "myapp",  # an app that contains the custom user model
```

Comment:
I can also revert the rename back to `"users"`, but then the template configuration example would read something like `users/users/create.html`.

Plus I wanted to sync the app name with the examples in the custom `GroupViewSet` docs, which previously used `"someapp"`.